scripts/opt-in/vscode.sh

```diff
@@ -0,0 +1,28 @@
+ME=`whoami`
+VSCODE_PREFERENCES_SOURCE_FOLDER="${MY_DIR}/files/vscode"
+VSCODE_PREFERENCES_DESTINATION_FOLDER="/Users/$ME/Library/Application Support/Code/User"
+
+rm -rf /tmp/"Visual Studio Code.app"
+rm -rf /tmp/vscode.zip
+
+curl -o /tmp/vscode.zip -L https://go.microsoft.com/fwlink/?LinkID=620882
```

Comment:
We can use `brew cask install visual-studio-code` to install vscode. Was that not an option?

Reply:
Nice. We can change it if that's a requirement, but I'm not personally a fan of using slower moving package managers when something is directly available from the source and doesn't require os specific install scripts.

Reply:
Cask is much easier to maintain (the open source community) and is constantly updated every time we run this script. It also handles the symlink for us.

What we lose in slowness from the package manager we gain in long-term maintainability, which is important for `workstation-setup`.